lib/measures/tbd/resources/psi.rb

```diff
@@ -962,25 +1392,64 @@ def processTBD(os_model, psi_set, io_path = nil, schema_path = nil, gen_kiva)
     type = :ceiling if /ceiling/i.match(s.surfaceType)
     type = :wall    if /wall/i.match(s.surfaceType)
 
-    ground   = s.isGroundSurface
-    boundary = s.outsideBoundaryCondition
     points   = (t * s.vertices).map{ |v| Topolys::Point3D.new(v.x, v.y, v.z) }
     minz     = (points.map{ |p| p.z }).min
 
     # Content of the hash will evolve over the next few hundred lines.
     surfaces[id] = {
-      type:     type,
-      ground:   ground,
-      boundary: boundary,
-      space:    space,
-      gross:    s.grossArea,
-      net:      s.netArea,
-      points:   points,
-      minz:     minz,
-      n:        n
+      type:         type,
+      conditioned:  conditioned,
+      ground:       ground,
+      boundary:     boundary,
+      space:        space,
+      gross:        s.grossArea,
```

Comment:
Only populating TBD opaque surfaces with API-reported gross area. For net area, relying on Frame & Divider-related workarounds.